### ISSUE TITLE
Compatibility problem

### ISSUE THREAD

**Issue by Namelessys** [max1220/lua-getch#4]
Source code written for older versions is incompatible with the newest version.

No big deal, but some versioning indicating that a new version breaks compatibility would be nice.
Maybe following the [Semantic Versioning](https://semver.org/) standard.

Have a nice day.

**Comment by max1220** [comment]
You're correct, previous versions didn't follow a clear versioning specification.

The project uses LuaRocks now, which includes versioning.
I've added the missing bits in the 0.0-1 rockspec now(see 6f6949a7d9d439a04662f39a340635991b68e733), including tagging the latest commit(new versions will get new rockspecs).
I've added some documentation on how to use LuaRocks for installation(9064f00ce39dc835fb34ba44d6b6dcf8c08516e2)
I've also added a scm rockspec that always uses the latest master from git.

(I'm marking this as closed, but if you have further comments, corrections etc. feel free to let me know :P)

Thank you for your contribution.
